docs/docs/cmd/teams/tab/tab-get.md

```diff
@@ -47,3 +47,38 @@ Get a Microsoft Teams Tab with name _Tab Name_
 ```sh
 m365 teams tab get --teamName "Team Name" --channelName "Channel Name" --name "Tab Name"
 ```
+
+## Response
+
+=== "JSON"
+
+    ``` json
+    {
+      "id": "34991fbf-59f4-48d9-b094-b9d64d550e23",
+      "displayName": "Polly",
```

Comment:
Let's try to align the output with the data used in the example. Same goes for text and csv.